includes/indices/class-algolia-index.php

```diff
@@ -673,13 +702,21 @@ public function de_index_items() {
 	 *
 	 * @return int
 	 */
-	public function get_re_index_batch_size() {
-		$batch_size = (int) apply_filters( 'algolia_indexing_batch_size', 100 );
+	protected function calculate_re_index_batch_size() {
+		$batch_size = (int) apply_filters( 'algolia_indexing_batch_size', 50 ); // Gentle: batch size 50.
```

Comment:
I'm still on the fence about changing this when we have a filter for it as well.

This is going to make the overall indexing process slower for those who may not have as huge of a site as I know we've been working on.

Reply:
Wasn't worried much about the public vs protected since you created a getter of the same name which will prevent breaking of things.

Reply:
@tw2113 resolving this comment now then :) I've changed the batch size back to 100.